src/Parser/GuardianDruid/Modules/Features/GalacticGuardian.js

```diff
@@ -60,6 +63,34 @@ class GalacticGuardian extends Module {
       }
     }
   }
+
+  suggestions(when) {
+    const unusedGGProcs = 1 - (this.consumedGGProc / this.GGProcsTotal);
+    
+    this.active &&
+    when(unusedGGProcs).isGreaterThan(0.3)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span>Your <SpellLink id={SPELLS.GALACTIC_GUARDIAN.id} /> procs should be used as soon as you get them so they are not overwritten.</span>)
```

Comment:
This suggestion may be unclear, normally I'd write what happened and the do the suggestion, e.g.

Your <SpellLink id={SPELLS.GALACTIC_GUARDIAN.id} /> proc usage can be improved. Try to use the procs as soon as you get them so they are not overwritten.
or
You wasted 20% of your <SpellLink id={SPELLS.GALACTIC_GUARDIAN.id} /> procs. Try to use the procs as soon as you get them so they are not overwritten.

Calling it "wasted" instead of "unused" might be better depending on how this works (I don't know).